doc/reference/tooling/tt_cli/cluster.rst

```diff
@@ -271,6 +272,74 @@ to ``ro`` and reloads the configuration.
     replica set members, so there can be any number of read-write instances in one replica set.
 
 
+.. _tt-cluster-failover:
+
+failover
+--------
+
+.. code-block:: console
+
+    $ tt cluster failover SUBCOMMAND [OPTION ...]
+
+``tt cluster failover`` manages a :ref:`supervised failover <repl_supervised_failover>` in Tarantool clusters.
+
+-   :ref:`switch <tt-cluster-failover-switch>`
+-   :ref:`switch-status <tt-cluster-failover-switch-status>`
+
+.. important::
+
+    ``tt cluster failover`` works only with centralized cluster configurations stored in etcd.
+
+
+.. _tt-cluster-failover-switch:
+
+switch
+~~~~~~
+
+.. code-block:: console
+
+    $ tt cluster failover switch URI INSTANCE_NAME [OPTION ...]
```

Comment:
Maybe it worth to mention `-w` here explicitly, because this option is likely often used together with this command.